web-common/src/features/dashboards/dashboard-stores.ts

```diff
@@ -518,4 +531,98 @@ export function useDashboardStore(
   });
 }
 
+/***
+ * Dervied stores to get time range and comparison range to be
+ * used for fetching data. If we have a scrub range and
+ * isScrubbing is false, use that, otherwise use the selected
+ * time range
+ */
+
+let lastScrubRange = { start: null, end: null };
```

Comment:
Why the global state here? Is that going to cause weird behavior when switching between dashboards?

Reply:
Fixed